shepherd.js/package.json

```diff
@@ -18,20 +18,19 @@
     "Robbie Wagner <rwwagner90@gmail.com>",
     "Chuck Carpenter <chuck@shipshape.io>"
   ],
+  "main": "dist/shepherd.js",
+  "module": "dist/shepherd.mjs",
   "exports": {
     ".": {
-      "types": "./dist/shepherd.d.ts",
-      "require": "./dist/shepherd.js",
       "import": "./dist/shepherd.mjs",
-      "default": "./dist/shepherd.js"
+      "require": "./dist/shepherd.js"
     },
     "./*": {
-      "types": "./dist/*.d.ts",
-      "require": "./dist/*.js",
       "import": "./dist/*.mjs",
-      "default": "./dist/*.js"
+      "require": "./dist/*.js"
     }
   },
+  "types": "shepherd.d.ts",
```

Comment:
I don't think we want this. We had all the types files in the dist before and we want them to correspond to the files people are importing.